README.md

```diff
@@ -1,37 +1,102 @@
-# Storybook Addon Visual Tests
+# Storybook Visual Testing addon
 
-Visual Testing addon with Chromatic
+The Visual testing addon enables you to run visual tests on your stories and compare changes with the latest baselines across multiple viewports and browsers to catch UI regressions early in development without leaving Storybook.
 
-## Installation
 
-Add the `@chromaui/visual-tests` addon, and add it to `main.js`:
+## Prerequisites
 
-```
-yarn add --dev @chromaui/visual-tests
-```
+- Chromatic [account configured](https://www.chromatic.com/docs/setup#sign-up) with access to a project
+- Storybook 7.2 or later
 
+## Getting Started
+
+1. Install this addon by adding the `@chromaui/addon-visual-tests` dependency:
+
+```shell
+yarn add --dev @chromaui/addon-visual-tests
 ```
-// main.js:
-"addons": ["@chromaui/visual-tests"]
-```
 
-### Troubleshooting
+2. Update your Storybook configuration file (e.g., `.storybook/main.js|ts`) to include the addon:
+
+```ts
+// .storybook/main.ts
+
+// Replace your-framework with the framework you are using (e.g., react-webpack5, vue3-vite)
+import type { StorybookConfig } from '@storybook/your-framework';
 
-If you see:
+const config: StorybookConfig = {
+  framework: '@storybook/your-framework',
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
+  addons: [
+    // Other Storybook addons
+    '@chromaui/addon-visual-tests',
+  ],
+};
 
+export default config;
 ```
```

Comment:
Should we just recommend people run `npx sb@latest add @chromaui/addon-visual-tests` for these two steps?

Reply:
I thought of it but didn't have the time to test it and see if the command works. Hence I went with this approach. I'll test it out shortly and if nothing stands out, I'll update this part with it

Reply:
Updated with the latest commit.